curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f7708379ef00668b843d4.md

```diff
@@ -0,0 +1,117 @@
+---
+id: 671f7708379ef00668b843d4
+title: Task 98
+challengeType: 19
+dashedName: task-98
+---
+
+<!-- (Audio) Maria: I remember you would often stay late, fine-tuning the code until it was just right. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+Did Brian use to work regular hours to adjust the code?
```

Comment:
```suggestion
Did Brian work regular hours to adjust the code?
```

Less clunky.

Reply:
This one is on purpose since the idea is to introduce 'used to' and its negative and interrogative forms.